components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -175,6 +176,12 @@ const ContributorHighlightCard = ({
     }
   };
 
+  const handleClickOutsideDropdownContent = (e: MouseEvent) => {
```

Comment:
Have we used this logic elsewhere yet @OgDev-01. I feel like I've seen this code before. Just mentioning it as it might make sense to promote this to a custom hook which could be done in a separate PR.

Reply:
Yes i have, I can quickly add that to this PR... Thanks for mentioning 👍